src/screens/PostScreen.tsx

```diff
@@ -7,37 +8,67 @@ import { styles } from './styles';
 export default function PostScreen() {
   return (
     <View style={styles.container}>
-      <ProfilePlaceholder />
-      <Text>rbeggs</Text>
-      <Text>September 19</Text>
-      <Text>
+      {/* Profile Header */}
```

Comment:
Consider applying styles to add spacing between the content and the sides of the screen. You can do so with the `padding` prop applied to the `container` View.